_data/gaming.yml

```diff
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
+    url: https://www.nexon.com/main/en
+    img: nexon.png
+    tfa:
+      - software
```

Comment:
```suggestion
      - proprietary
```

Reply:
It appears to be a generic TOTP (Google Authenticator based), however. I registered it and get generic 6-number codes.

Reply:
The documentation page talks about "The Authenticator" so I thought it was some kind of app that they had build.
Which 2FA app(s) have you tried with Nexon?

Reply:
I set it up using Authy, which provides the aforementioned 6-number codes. Google Authenticator is explicitly recommended; you can add accounts using the provided QR code or a manual code. I have attached images to reference these statements.

![Screenshot 2019-10-16 at 18 58 35](https://user-images.githubusercontent.com/31264514/66971807-61765500-f047-11e9-978f-70cd3af0c89e.png)
![Screenshot 2019-10-16 at 18 59 15](https://user-images.githubusercontent.com/31264514/66971814-676c3600-f047-11e9-8278-bd582b6caf94.png)

Reply:
Alright. Thanks!
Here's a better fitting change then:
```suggestion
      - totp
```